docs/source/conf.py

```diff
@@ -69,9 +69,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "0.14"
+version = "0.13"
```

Comment:
```suggestion
import openmc
version = '.'.join(openmc.__version__.split('.')[0:2])
```

Reply:
I'm not sure importing the package within conf.py will work smoothly, so I'd rather not take a chance at messing up our documentation on release. We can try this out in the future though!